standards/decommissioning.md

```diff
@@ -0,0 +1,12 @@
+# Decommissioning and Deprecating Apps
+
+NYPL creates different applications for user interfaces, data pipeline flow, APIs, and more. While we maintain these applications, sometimes it's better to move from one platform to another for many reasons.
+
+When this happens, the developer(s) working on the new iteration of an application MUST let others know that the older version will be deprecated. This can reduce confusion on whether packages need to be updated for an app or codebase.
+
+The above also applies to codebases that will no longer be used, such as moving code from one codebase to another, or upgrading to a different tool.
+
+Developer(s) MUST:
+* update the codebase's README with a deprecated and a no longer supported note
+* let the team know that an app/codebase is deprecated
+* update related documentation where the app/codebase's name comes up
```

Comment:
This one is trickier... how should they find documentation? Obviously should be in Confluence, but all repos too? If in repos we should specify that it must be via PR (not just updating, e.g., master) or else Jira tickets should be opened for teams to update their own repos. I agree there needs to be a docs update step though.

Reply:
Whenever a package on npm is deprecated, you usually don't find out until you look for its update. Maybe it's okay to not contact any repos using it (because we also don't know all the repos using it). It's also possible that the latest version they are using is stable enough or their purposes so maybe opening up Jira tickets isn't absolutely necessary.